sky/server/server.py

```diff
@@ -1085,6 +1085,10 @@ async def complete_storage_name(incomplete: str,) -> List[str]:
                     host=cmd_args.host,
                     port=cmd_args.port,
                     workers=num_workers)
+    except Exception as exc:  # pylint: disable=broad-except
+        logger.error(f'Failed to start SkyPilot API server: '
+                     f'{common_utils.format_exception(exc, use_bracket=True)}')
```

Comment:
Should we re-raise the exception? Otherwise, the program will end successfully?
```suggestion
    except Exception as exc:  # pylint: disable=broad-except
        logger.error(f'Failed to start SkyPilot API server: '
                     f'{common_utils.format_exception(exc, use_bracket=True)}')
        raise
```

Reply:
Good catch